extensions/procfs/SystemClockDuration.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <sys/param.h>
```

Comment:
`HZ` is defined in `asm-generic/param.h`, which is (indirectly) included from `sys/param.h` in Ubuntu 20.04, but not in the alpine:3.13 docker image.  Including `asm-generic/param.h` directly works on both platforms.
```suggestion
#include <asm-generic/param.h>
```